src/views/landing.njk

```diff
@@ -37,4 +36,13 @@
       <p class="govuk-body-s">Read the <a href="https://www.gov.uk/government/publications/company-strike-off-dissolution-and-restoration/strike-off-dissolution-and-restoration" target="_blank" class="govuk-link">guidance for striking off and dissolving a company (opens in a new tab)</a>.</p>
     </div>
   </div>
+  {% if piwik %}
+    <script nonce={{ nonce | dump | safe }}>
+      $(document).ready(function() {
+        $('#start-now').on('click', function() {
+          _paq.push(["trackGoal", {{ piwik.landingPageStartGoalId }}]);
+        });
+      })
+    </script>
+  {% endif %}
```

Comment:
could we make this a partial possibly? Something like:
```
{% set goalId = piwik.landingPageStartGoalId %}
{% set selector = "#start-now" %}
{% include components/piwik-click-listener.njk %}
```

and where the partial contains:
```
{% if piwik %}
    <script nonce={{ nonce | dump | safe }}>
      $(document).ready(function() {
        $({{ selector }}).on('click', function() {
          _paq.push(["trackGoal", {{ goalId }}]);
        });
      })
    </script>
  {% endif %}
```